docusaurus/docs/how-to-guides/ui-extensions/register-an-extension.md

```diff
@@ -13,23 +13,23 @@ sidebar_position: 20
 
 import ExtensionPoints from '@shared/extension-points.md';
 
-An _extension_ is a plugin-defined link or a React component that is rendered in either the core Grafana UI or in another app plugin.
+A UI extension is either a plugin-defined link or a React component that you can render in the core Grafana UI or in another app plugin. 
```

Comment:
There's even another twist to this now: an extension can also be a generic function (which doesn't necessarily render anything, just returns a value based on a list of arguments). Due to this we started to just call them "extensions", as we thought it's less misleading. (I know, right now everything is misleading in here :))